master/internal/experiment/authz_iface.go

```diff
@@ -93,7 +93,7 @@ type ExperimentAuthZ interface {
 
 	// POST /api/v1/experiments
 	CanCreateExperiment(
-		ctx context.Context, curUser model.User, proj *projectv1.Project, e *model.Experiment,
+		ctx context.Context, curUser model.User, proj *projectv1.Project,
```

Comment:
Huh, I'm not sure what the original intent was behind having an experiment be a parameter of this interface, but this does seem like a good change.